extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -103,6 +103,7 @@
   opacity: 0 !important;
   transition: transform var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve) !important;
   transform: translateX(60px) !important;
+  flex-shrink: 0 !important;
```

Comment:
Mostly asking because it's completely unrelated to the PR, is this needed?

Reply:
Oh! I found that the percentage value on a poll sometimes would be on top of the percentage symbol, I could do a separate PR for it but thought to add it as I go, as it's something that found not very often (only when the option text is long enough). Disallowing the shrink (setting to 0) makes the component take all the space needed. I attached an image of the "before", after this it gets fixed.
![image](https://user-images.githubusercontent.com/22420856/88438001-acdb4200-cdd5-11ea-9a6b-050cb9348101.png)